test/gtest/uct/ib/test_sockaddr.cc

```diff
@@ -1081,6 +1085,22 @@ UCT_INSTANTIATE_SOCKADDR_TEST_CASE(test_uct_sockaddr_stress)
 
 
 class test_uct_sockaddr_multiple_cms : public test_uct_sockaddr {
+    class scoped_async_lock {
```

Comment:
maybe to test_helpers.h

Reply:
we already have common async_lock for entity class, this one is specific for this test only, related to migration conn_request between CMs